versions.yaml

```diff
@@ -29,6 +29,28 @@ _variables:
     gpg: '08a0a054aecedec4aa2772569325a346403d2c1fba6f88739553416c2f4f77e6'
 
 versions:
+  - dir: '1/debian9/1.18'
```

Comment:
As discussed, the latest stable nginx is 1.20.1

Reply:
updated to 1.20.1 with "buster" repo on Debian10